specification/dartLangSpec.tex

```diff
@@ -21684,7 +23732,8 @@ \subsection{Type Void}
 must compare equal (according to the \lit{==} operator \ref{equality})
 to any instance of \code{Type} which reifies the type \code{Object}
 (\ref{dynamicTypeSystem}).
-It is not guaranteed that \code{identical(\VOID, Object)} evaluates to true.
+It is not guaranteed that \code{identical(\VOID, Object)} evaluates to
+the \TRUE{} object.
```

Comment:
I would expect `identical(void, Object)` to be `false`, actually. Maybe the text should mention `Object?` instead of the `Object`?